Sources/ForageSDK/Component/ForagePINTextField/Vault/RosettaPINTextField.swift

```diff
@@ -0,0 +1,175 @@
+//
+//  RosettaPINTextField.swift
+//
+//
+//  Created by Evan Freeze on 6/6/24.
+//  Copyright © 2024-Present Forage Technology Corporation. All rights reserved.
+//
+
+import Combine
+import UIKit
+
+class ForageTextFieldWrapper: UIView, VaultWrapper, UITextFieldDelegate {
```

Comment:
```suggestion
class RosettaPINTextField: UIView, VaultWrapper, UITextFieldDelegate {
```

Reply:
ah yeah meant to do this before 🤦🏻‍♂️ updated in [b9b7bc4](https://github.com/teamforage/forage-ios-sdk/pull/169/commits/b9b7bc4eb3a1a63150546f4118c40446f90d458d)